README.md

```diff
@@ -2,37 +2,17 @@
 
 UK Gov dash components is a Dash component library.
 
-Get started with:
-1. Install Dash and its dependencies: https://dash.plotly.com/installation
-2. Run `python usage.py`
-3. Visit http://localhost:8050 in your web browser
-
-## Contributing
-
-See [CONTRIBUTING.md](./CONTRIBUTING.md)

```

Comment:
Shall we remove the `CONTRIBUTING.md` file too?  It doesn't look very valuable when I had a glance.

Reply:
Good idea.